docs/core/tools/custom-templates.md

```diff
@@ -91,6 +91,62 @@ The *template.json* file looks like the following:
 
 The *mytemplate* folder is an installable template package. Once the package is installed, the `shortName` can be used with the `dotnet new` command. For example, `dotnet new adatumconsole` would output the `console.cs` and `readme.txt` files to the current folder.
 
+### Template localization
+The .NET templates are localizable. If a template is localized for the language matching the current locale, its elements will appear in the same language as the CLI. Localization is optional when creating new templates.
+
+The localizable elements on a template are:
+- Name
+- Author
+- Description
+- Symbols
+  - Description
+  - Display Name
+  - Descriptions and Diplay name of choices for choice paramemters 
+- Post actions
+  - description
+  - Manual instructions
+
+Localization files have a JSON format, and just one file per culture should exist. The naming convention is: `templatestrings.<lang code>.json`, where `lang code` corresponds to one of the [CultureInfo](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.name) options. All localization files should be inside the `.template-config\localize` folder.
+
+The localization JSON consists of key value pairs:
+- The key is the reference to an element of `template.json` to be localized. If the element is a child, use the full path with a `/` delimiter.
+- The value is the localization string of the element given by the key.
+
+For more information about localizing templates, see the [dotnet templating wiki's localization page](https://github.com/dotnet/templating/wiki/Localization).
+
+#### Example
+For example, here's *template.json* file with some localiable fields:
+```JSON
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": "Config",
+  "name": "EditorConfig file",
+  "description": "Creates an .editorconfig file for configuring code style preferences.",
+  "symbols": {
+    "Empty": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "displayName": "Empty",
+      "description": "Creates empty .editorconfig instead of the defaults for .NET."
+    }
+  }
+}
+```
+
+And some fields are to be localizes to Brazilian Portuguese. The filename will be `templatestrings.pt-BR.json` to match the culture, and it would look like:
```

Comment:
```suggestion
And some fields are to be localized to Brazilian Portuguese. The filename will be `templatestrings.pt-BR.json` to match the culture, and it would look like:
```